src/SiteExtensions/LoggingAggregate/build.cmd

```diff
@@ -1,3 +1,3 @@
 @ECHO OFF
-SET RepoRoot="%~dp0..\..\.."
-%RepoRoot%\build.cmd -LockFile %RepoRoot%\korebuild-lock.txt -Path %~dp0 %*
+SET RepoRoot=%~dp0..\..\..
+%RepoRoot%\build.cmd -LockFile %RepoRoot%\korebuild-lock.txt -projects %~dp0**\*.*proj %*
```

Comment:
IIRC, this code still requires using `-path` not `-projects`. We didn't completely backport all the build.cmd changes to 2.x branches.

Reply:
Wasn't working correctly for me until I made these changes :/

Reply:
I suspect you need the first change (remove the quotes), but not sure about the second. src/SiteExtensions/LoggingAggregate/ still has its own 'subfolder' of KoreBuild stuff, like 'src/SiteExtensions/LoggingAggregate/build/' which would only be imported when using `-path`.

Reply:
`C:\Users\Brennan\.dotnet\buildtools\korebuild\2.2.1-build-20190318.1\KoreBuild.Common.targets(81,5): error MSB3191: Unable to create directory "E:\ASPNETCORE\aspnetcore\src\SiteExtensions\LoggingAggregate\"\artifacts\".`

Reply:
Looks a like a problem with an extra trailing backslash. How did this work before, i wonder?

Reply:
No idea, it's all new to me

Reply:
I still suspect this will fail, but if you validated it, :shipit: